lib/src/signed_video_h26x_sign.c

```diff
@@ -574,6 +574,12 @@ signed_video_add_nalu_part_for_signing_with_timestamp(signed_video_t *self,
         // adding it to the SEI NALU. Now it is time to start a new GOP, that is, hash and add this
         // first NALU of the GOP.
         SV_THROW(hash_and_add(self, &nalu));
+        // Chek if Nalu has been used for linking before.
+        if (!nalu.used_for_linked_hash) {
```

Comment:
Is the member `used_for_linked_hash` really needed. The only place where it is changed is here and the same NALU can never end up here again.
Also, please explain why you are doing this in the code the reflects the "old" code and not in the "new" code if `self->linked_hash_on`